turbo.json

```
@@ -13,6 +13,9 @@
       "dependsOn": ["generate"],
       "cache": false
     },
+    "attw": {
+      "dependsOn": ["format"]
```

`attw` runs `npm pack`, so it needs the build artifacts. It seems very odd to rely on the build being run through two levels of dependencies. Can't we simply do

```suggestion
      "dependsOn": ["build"]
```

here?